test/StoragelessTest/Http/SessionMiddlewareTest.php

```diff
@@ -558,14 +592,14 @@ public function testShouldNotRegenerateTokenWhenRequestHasATokenThatIsFarFromExp
     public function validMiddlewaresProvider(): array
     {
         return $this->defaultMiddlewaresProvider() + [
-            [
+            'from-constructor' => [
```

Comment:
HUGE bug introduced [here](https://github.com/psr7-sessions/storageless/pull/119/files#diff-f49a37ecc9fa0480953b1a1534d1cc67755949456a3ad63c07b212e1958359edR556) by myself, I'm so sorry: without a non-numeric key, the `+` operator n arrays overwrite the same-numeric-key entries, thus removing one provider.

Weird that this bug wasn't captured by mutation testing though

Reply:
Is the bug from #119 also fixed in current stable? I don't remember seeing a fix :|

Reply:
No, would you like a separate PR for stable branch?

Reply:
Yes please!  If it's broken, we should fix it :|